example/helloWorld/CMakeLists.txt

```diff
@@ -34,11 +34,11 @@ project(${_TARGET_NAME} LANGUAGES CXX)
 # Find alpaka.
 
 if(NOT TARGET alpaka::alpaka)
-    option(USE_ALPAKA_SOURCE_TREE "Use alpaka's source tree instead of an alpaka installation" OFF)
+    option(USE_alpaka_SOURCE_TREE "Use alpaka's source tree instead of an alpaka installation" OFF)
```

Comment:
Nothing for this PR but should this maybe named `alpaka_USE_SOURCE_TREE` to group all alpaka variables if you use `ccmake` or `cmake -LA .`